src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php

```diff
@@ -137,6 +137,8 @@ private function createProvinceCodeChoiceForm(CountryInterface $country, $provin
         return $this->formFactory->createNamed('provinceCode', ProvinceCodeChoiceType::class, $provinceCode, [
             'country' => $country,
             'auto_initialize' => false,
+            'label' => 'sylius.form.address.province',
```

Comment:
Are these really needed? These are the default values, both for `label` and `placeholder` (ProvinceCodeChoiceType is a child of ProvinceChoiceType defining those).